packages/sheets-ui/src/services/selection/selection-control.ts

```diff
@@ -144,7 +143,7 @@ export class SelectionControl extends Disposable {
      * Observer: prompt.controller
      */
     readonly selectionMoving$ = new Subject<IRangeWithCoord>();
-    readonly selectionMoved$ = new Subject<IRangeWithCoord>();
```

Comment:
make sure , this one is no used in pro,  otherwise you should rename it both two project